src/ScriptCs.Hosting/ScriptServicesBuilder.cs

```diff
@@ -131,6 +133,18 @@ public IScriptServicesBuilder Debug(bool debug = true)
             return this;
         }
 
+        public IScriptServicesBuilder SetOverride<TContract, TImpl>(TImpl value) where TImpl : TContract
+        {
+            Overrides[typeof(TContract)] = value;
+            return this;
+        }
+
+        public IScriptServicesBuilder SetOverride<TContract, TImpl>() where TImpl : TContract
+        {
+            Overrides[typeof(TContract)] = typeof(TImpl);
```

Comment:
@glennblock how does this work? Is it possible to set not only a value for a given contract type (as in the above overload) but also just an implementation type?